README.md

```diff
@@ -19,6 +19,15 @@ Part of the [Equinor Subsurface Visualization project](https://github.com/equino
 Using npm:
 ```bash
 npm install --save @equinor/esv-intersection
+
+To run application in your machine :
+
+$ git clone git@github.com:equinor/esv-intersection
+$ cd esv-intersection
+$ git submodule init
+$ git submodule update
+$ npm run storybook
+
```

Comment:
This should not hijack the code block cell for installing using npm, it should also not come in between installing the different ways of installing the package. It would be better if it came in at the end of the installation subheader